plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoClientConfig.java

```diff
@@ -51,6 +51,7 @@ public void testExplicitPropertyMappings()
     {
         Map<String, String> properties = ImmutableMap.<String, String>builder()
                 .put("mongodb.schema-collection", "_my_schema")
+                .put("mongodb.schema-database", "_my_schema_db")
```

Comment:
Please add a new test class extending `AbstractTestQueryFramework` and use `mongodb.schema-database` in the test.